testing/src/scenario/context.py

```diff
@@ -600,8 +600,8 @@ def test_emitted():
     charm_root: str | pathlib.Path | None
     """The charm root directory to use when executing the charm.
 
-    Before running the event, the charm's ``/src``, any libs, and the metadata, config, and action
-    YAML is copied to the charm root. If ``charm_root`` is None, then a temporary directory is
+    Before running the event, the charm's metadata, config, and action YAML is
```

Comment:
my 2c: either "YAML file", or list the actual file names explicitly, like `metadat.yaml`.